guides/api.md

```diff
@@ -674,6 +664,32 @@ logger.info(ao.insertLogObject({
     more: 'there will be an added ao property'
 }))
 ```
+<a name="ao.wrapLambdaHandler"></a>
+
+### ao.wrapLambdaHandler([handler]) ⇒ <code>function</code>
+Wrap the lambda handler async function so it can be traced by AppOptics APM.
```

Comment:
Hmm I had the impression that you got the wrapper to work with both the "async function"-based AWS lambda handler, and the "callback"-based handler... but now reading above and the KB draft, is that true or does it only support the async-function-based handler?

Reply:
it works with both.

Reply:
Ah good -- I find the "lambda handler async function" confusing then, how about just saying "lambda handler function" in line 670 then.